pycloudlib/oci/cloud.py

```diff
@@ -217,6 +223,7 @@ def launch(
         instance_type="VM.Standard2.1",
         user_data=None,
         wait=True,
+        retry_strategy=None,
```

Comment:
Being this an actual keyword-only argument (because it is not defined as an explicit arg in the base class, therefore it belongs to the **kwargs interface and it is captured here), I wonder if we should enforce it:

```suggestion
        *,
        retry_strategy=None,
```

By doing so, we would prevent client-code consuming this function like:

```python
oci.launch(image_id, instance_type, user_data, wait, "my_cool_retry_strategy")
```

which kind of break the Liskov Substitution Principle, as instances of other child classes do not have to have a fifth positional argument.

More info: [PEP 3102 – Keyword-Only Arguments](https://peps.python.org/pep-3102/)